py3status/i3status.py

```diff
@@ -248,6 +250,11 @@ def write_tmp_i3status_config(self, tmpfile):
             section = self.config[section_name]
             self.write_in_tmpfile('%s {\n' % section_name, tmpfile)
             for key, value in section.items():
+                # don't include color values
```

Comment:
looking at the i3status docs the modules themselves do not accept color_bad etc

or is this just about a poor comment?